index.html

```diff
@@ -116,7 +129,56 @@ <h2 class="recent-work">My Recent Works</h2>
             </div>
           </div>
           <div class="see-project">
-            <button class="invite one">See Project</button>
+            <button class="invite one" id="invite">See Project</button>
+          </div>
+          <div class="pop-up">
+            <div class="multi-mid">
+              <div>
+                <h3 class="pop-title">Multi Post Stories</h3>
+                <h3 class="pop-desk-title">
+                  Keeping track of hundreds of components website
+                </h3>
+              </div>
+              <div>
+                <img id="xx" alt="cancel" src="icons/Icon.png" />
+              </div>
+            </div>
+            <div class="instrument">
+              <div class="js-pop">
+                <ul class="puOut">
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby on rails</li>
+                </ul>
+              </div>
+            </div>
+            <div class="pop-desk">
+              <div class="pop-img">
+                <img
+                  class="snap"
+                  src="icons/Snapshoot-Portfolio.png"
+                  alt="snapshoot"
+                />
+              </div>
+              <div class="desk-msgs">
+                <p class="msgs">
+                  Lorem Ipsum is simply dummy text of the printing and
+                  typesetting industry. Lorem Ipsum has been the industry's
+                  standard dummy text ever since the 1500s, when an unknown
+                  printer took a galley of type and scrambled it to make a type
+                  specimen book. It has survived not only five centuries, but
+                  also the leap into electronic typesetting, remaining essent
+                </p>
+                <div class="live-source">
+                  <button>
+                    See Live <img src="icons/live.png" alt="live icon" />
+                  </button>
```

Comment:
- [ ] Please align buttons at the same level as the image bottom:

Yours  | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/151184637-c667387d-8697-4dfc-9943-b32ede45424c.png)  | ![image](https://user-images.githubusercontent.com/77038610/151184779-d524515c-2bf5-4bb3-be39-4034e4abd629.png)

- [ ] Please try to match the proportions of left and right side content as provided in Figma, eg. in  Figma, the left side image is taking 60% of the space, in your case it is taking around 45%.
- [ ] On hover buttons are having a white background and the `see live` & `github` icons are becoming invisible, please make sure to fill them with orange color as well as the text.